$NetBSD$

gcc   -I../Include -DDINT -c amd_aat.c -o amd_i_aat.o
In file included from ../Include/amd_internal.h:157,
                 from amd_aat.c:18:
../Include/amd.h:46:10: fatal error: SuiteSparse_config.h: No such file or directory
   46 | #include "SuiteSparse_config.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** Error code 1

Stop.
make[2]: stopped in /tmp/math/R-Matrix/work/Matrix/src/AMD/Source

--- src/AMD/Source/Makefile.orig	2023-07-03 22:28:32.000000000 +0900
+++ src/AMD/Source/Makefile	2023-08-12 07:22:56.796785743 +0900
@@ -1,4 +1,5 @@
 PKG_CPPFLAGS = -I../Include -I../../SuiteSparse_config
+ALL_CFLAGS = -I../Include -I../../SuiteSparse_config
 
 LIB = ../../AMD.a
 
